Implementation of new frontend interface for KYC validator

adamsnows opened the pull request:
### **Enhanced interface and internationalization for KYC Validator**
This PR implements significant improvements to the KYC Face Validator interface, adding multi-language support and enhancing the user experience. Building upon the previous redesign, this update brings even more functionality and refinement.

### **Previous Redesign Features**

- Responsive design using Tailwind CSS with light/dark theme support
- Intuitive image upload interface with immediate visual feedback
- Image preview area for better viewing before comparison
- Image loading indicator during validation processing
- Clear comparison results with visual match indicators
- Expandable debug section for easier development and testing
- Proper error handling with visual feedback for the user

### **New Internationalization Features**

- Addition of language toggle buttons with flags for Brazil (PT) and United States (EN)
- Complete interface translation for Portuguese and English
- Selected language sent to backend via Accept-Language header
- Support for localized error messages in API responses
- Code Organization Improvements
- Separation of JavaScript from HTML into a dedicated file (/js/main.js)
- Moving inline styles to the global Tailwind CSS file
- Creation of custom Tailwind components for reusable elements
- Enhanced documentation with JSDoc comments in JavaScript code

### **Visual Enhancements**

- Improved JSON formatting in debug section with colored syntax highlighting
- Color scheme adaptation for both light and dark modes
- More consistent organization of UI components
- Enhanced visual feedback for language selection

### **Documentation**

- Updated README with instructions about the internationalization feature
- Details about custom Tailwind CSS components
- Updated project execution instructions

juninhopo commented:
Remove animation on click to compare

![Screenshot 2025-05-07 at 00 05 10](https://github.com/user-attachments/assets/7c6ec388-7685-40b8-b50f-23e7cf3d8abb)

juninhopo commented:
Need to maintain the structure of the Identa Debug

after: ![Screenshot 2025-05-07 at 00 05 06](https://github.com/user-attachments/assets/2710646d-8bb9-45a9-bcef-613a55f6de25)
before: ![Screenshot 2025-05-07 at 00 06 06](https://github.com/user-attachments/assets/61186dd6-3baf-4691-80cf-ba1332903fed)

adamsnows commented:
> Remove animation on click to compare
> 
> ![Screenshot 2025-05-07 at 00 05 10](https://private-user-images.githubusercontent.com/26558998/441056973-7c6ec388-7685-40b8-b50f-23e7cf3d8abb.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDY1OTIxODgsIm5iZiI6MTc0NjU5MTg4OCwicGF0aCI6Ii8yNjU1ODk5OC80NDEwNTY5NzMtN2M2ZWMzODgtNzY4NS00MGI4LWI1MGYtMjNlN2NmM2Q4YWJiLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTA1MDclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwNTA3VDA0MjQ0OFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTg1N2JmOGYyMjY1NWMxMjM4MDJlNzk1N2NhNGUxMWIwMGNhZjBjNjVlMDk2OWZhMTJlYzYxY2Y0MWM5YzBjMTgmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.-YEgxvzXYUNRp8mwPQW8aqph4Hjds0ksKndT6IasAdw)

removed

adamsnows commented:
> Need to maintain the structure of the Identa Debug
> 
> after: ![Screenshot 2025-05-07 at 00 05 06](https://private-user-images.githubusercontent.com/26558998/441057150-2710646d-8bb9-45a9-bcef-613a55f6de25.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDY1OTIxODgsIm5iZiI6MTc0NjU5MTg4OCwicGF0aCI6Ii8yNjU1ODk5OC80NDEwNTcxNTAtMjcxMDY0NmQtOGJiOS00NWE5LWJjZWYtNjEzYTU1ZjZkZTI1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTA1MDclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwNTA3VDA0MjQ0OFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTczNGM5NDA0MzFkOTEwNTBmNzE1M2QzNDYzNjVmYTg5ODA3ZjM5YzdmOWI3ZDdkODMxY2IzYWJiYTMzMGEwMDImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.w4_7jbS7OyTiAHw6RTmSxrO5njGkqiSsw-_7P7qkhis) before: ![Screenshot 2025-05-07 at 00 06 06](https://private-user-images.githubusercontent.com/26558998/441057169-61186dd6-3baf-4691-80cf-ba1332903fed.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDY1OTIxODgsIm5iZiI6MTc0NjU5MTg4OCwicGF0aCI6Ii8yNjU1ODk5OC80NDEwNTcxNjktNjExODZkZDYtM2JhZi00NjkxLTgwY2YtYmExMzMyOTAzZmVkLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTA1MDclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwNTA3VDA0MjQ0OFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTM4MWFhMTJjOGEzYjFmNGE5YmYyODliMWFjOGEwNmY3OWEzZTIwMjg3Y2U0NDg2OTU0NDQwY2ZiOGRmMjFmYzkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.LFBx_2W4gCxrx-SmuRU1J_64FN4wwEC2nXEFFXbHNRI)

now added json view for response

adamsnows commented:
### **Internationalization Features**

- Addition of language toggle buttons with flags for Brazil (PT) and United States (EN)
- Complete interface translation for Portuguese and English
- Selected language sent to backend via Accept-Language header
- Support for localized error messages in API responses

### **Code Organization Improvements**

- Separation of JavaScript from HTML into a dedicated file (/js/main.js)
- Moving inline styles to the global Tailwind CSS file
- Creation of custom Tailwind components for reusable elements
- Enhanced documentation with JSDoc comments in JavaScript code

### **Visual Enhancements**

- Improved JSON formatting in debug section with colored syntax highlighting
- Color scheme adaptation for both light and dark modes
- More consistent organization of UI components
- Enhanced visual feedback for language selection

### **Documentation**

- Updated README with instructions about the internationalization feature
- Details about custom Tailwind CSS components
- Updated project execution instructions
- These improvements make the application more accessible to users of different languages and easier to maintain for developers, following best practices in code organization and interface design.

sibelius reviewed (approved):
what about nextjs app dir ?

juninhopo commented:
@sibelius dont need, the front is for only demo, dont need frontend

The real focus is the process decision by two images

juninhopo commented on src/utils/upload.ts:
```diff
@@ -5,22 +5,22 @@ import { fileTypeFromBuffer } from 'file-type';
+  storage: multer.memoryStorage(),
   limits: {
-    fileSize: 5 * 1024 * 1024, // 5MB max file size
+    fileSize: 10 * 1024 * 1024,
```
change to 5
